server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java

```diff
@@ -236,4 +285,13 @@ public Invocations getInvocations() {
       return completedFuture(responseBuilder.build());
    }
 
+   class CacheFullDetail {
+      public Stats stats;
+      public int size;
+      public Configuration configuration;
+      public List<Address> members;
+      public boolean rehashInProgress;
+      public MemoryConfiguration memory;
```

Comment:
This is incorrect. It should be the full cache configuration here. And it should be converted to JSON before passing it through jackson, otherwise it will use reflection and will print funny things (probably that's what's happening with the response in the docs)
Here's how to convert the config into JSON:  https://github.com/infinispan/infinispan/blob/38ba1de11f8b502087261c925aff08a014bda6d8/server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java#L199-L205

Reply:
Once you have the cache config as JSON, in your response object, you annotate with RAW json so that jackson will simply pass it to the output:
```java
 class ResponsePOJO {
      @JsonRawValue
      public Object getConfiguration() {
         return configuration;
      }
}
```

Reply:
I deleted that for now